### PR TITLE
LS-184-helm-doc

### PR DESCRIPTION
**Pull request by mendonk**
First draft

### CODE REVIEW

File: modules/ROOT/pages/helm-chart-config.adoc [hunk 1]
```diff
@@ -0,0 +1,77 @@
+= Helm Chart Configuration Values
+
+This page will explain configuration values included in the Helm chart https://github.com/datastax/pulsar-helm-chart/blob/master/helm-chart-sources/pulsar/values.yaml[values file].
+
+You can override chart configurations during installation as necessary if you’re an advanced user, or make changes after a default installation using `helm upgrade` at a later time.
```
Comment (jgillenwater) [inline modules/ROOT/pages/helm-chart-config.adoc:5]:
```suggestion
This page explains configuration values included in the Helm chart https://github.com/datastax/pulsar-helm-chart/blob/master/helm-chart-sources/pulsar/values.yaml[values file].

If you are an advanced user, you can override chart configurations during installation as necessary or make changes after a default installation using `helm upgrade` at a later time.
```

File: modules/ROOT/pages/helm-chart-config.adoc [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+|*Type*
+|*Default*
+|*Description*
+
+// | Key | Type | Default | Description |
+// |-----|------|-------------|------------------|
+// #| cassandra.enabled | bool | `true` | Enables installation of Cassandra cluster. Set to false if you only wish to install operators. |
+| fullnameOverride | string | `pulsar` | Overrides the name of the helm deployment 
+| dnsName | string | `pulsar.example.com` | DNS name for load balancer 
+| rbac | bool | `true` | use `ClusterRole` and `ClusterRoleBinding` resources when set to `true``, use namespace Role and RoleBinding resources when set to `false`
```
Comment (jgillenwater) [inline modules/ROOT/pages/helm-chart-config.adoc:27]:
```suggestion
| rbac | bool | `true` | Uses `ClusterRole` and `ClusterRoleBinding` resources when set to `true`; uses `NamespaceRole` and `RoleBinding` resources when set to `false` 
```

**Review by jgillenwater — CHANGES_REQUESTED** [review]
I think the structure is good, but the descriptions are documented inconsistently. Some state the key name to start the description. Some use more active language in the verb. Please edit for consistency and submit for review again.

**Comment by mendonk** [comment]
This will be doc'd as part of a larger effort in this repo, so this PR is no longer useful.

**Pull request closed without merging**